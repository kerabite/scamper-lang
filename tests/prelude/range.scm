(range 10)

(range 50)

(range 0)

(range -1)