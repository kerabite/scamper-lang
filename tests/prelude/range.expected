(list 0 1 2 3 4 5 6 7 8 9)
(list 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49)
null
:1:6: Runtime error:
    range expects that argument 1 obeys the following property: non-negative, but -1 was given.
    In program: (range -1)
